README.md

```diff
@@ -1,21 +1,19 @@
-## Development Setup
+# Studypal
 
-1. After cloning the project, install the dependencies.
+Studypal is an application where tutors can sign up to provide students with homework help in a free and convenient way.
 
-    ```bash
-    npm i
-    ```
+<img src="https://i.imgur.com/iiDjJVb.png">
 
-2. Setup firebase by generating a private key in the console and storing the file as `firebase_key.json` inside `/keys` directory.
+## Technologies Used
 
-3. Run the development server.
+Javascript • Firebase • Next.js • Node.js • Passage Auth • Tailwind CSS
 
-    ```bash
-    npm run dev
-    ```
+## Getting Started
 
-## Deploy on Vercel
+Check out Studypal here: https://studypal.support/
```

Comment:
Sorry @egwilborn! I removed the deployment already so this is a dead link 🙇‍♀️  Perhaps you could link to the youtube instead?

Reply:
@sirbully  That make sense! Sounds good I'll switch it to the Demo video. Thanks!